src/main/java/uk/ac/ebi/eva/evaseqcol/entities/SeqColLevelOneEntity.java

```diff
@@ -30,11 +35,20 @@ public class SeqColLevelOneEntity extends SeqColEntity{
     @Id
     @Column(nullable = false)
     @Enumerated(EnumType.STRING)
-    private NamingConvention namingConvention;
+    protected NamingConvention namingConvention;
 
-    public SeqColLevelOneEntity(String digest, JSONLevelOne jsonLevelOne, NamingConvention namingConvention){
-        super(digest);
+    public SeqColLevelOneEntity(String digest, NamingConvention namingConvention, JSONLevelOne jsonLevelOne){
+        super(digest, namingConvention);
         this.object = jsonLevelOne;
```

Comment:
Could we avoid attribute names like `object` and prefer a more descriptive attribute name like `seqColLevel1Object` ?

Reply:
Yes, `seqColLevel1Object` will be a better attribute/column name.
I'll rename as such.